build/misc.js

```diff
@@ -55,6 +55,7 @@ function getLocals () {
     const out = {
         _,
         orderObject,
+        log: console.log,
```

Comment:
Intentional, It does no harm being there and my local copy keeps getting it deleted by stash and branch operations.